src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,47 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int, ring::Union{MPolyRing, Nothing} = nothing)
+
+Given an ambient dimension and a subspace dimenson dimension return the ideal
+generated by the plucker relations. If the ring is not specified return the ideal in the multi variate polynomial ring over the rationals.
```

Comment:
```suggestion
generated by the plucker relations. If the ring is not specified return the ideal
 in the multi variate polynomial ring over the rationals.
```
Please avoid lines longer than ~90 chars (I usually aim for more like 80, but I am fine either way, as long as I can read it in the GitHub review view without having to scroll horizontally :-)